### DISCUSSION TITLE
search not working in custom attribute of type references

### DISCUSSION THREAD

**Discussion by ghost** [saleor/saleor#12009] [Q&A]
i have created a product attribute of type references with relation to products and when i click on assign references
![Screenshot from 2023-02-14 14-11-47](https://user-images.githubusercontent.com/83389170/218683599-d263e782-e66f-48f0-a10c-e7081132249e.png)
screen appears with all products present, i want to make search with specific product type but search is not working 
it is given that i can search using product name, attribute, product type etc, but this search is working only on product name/description, not working on product type as shown below
![Screenshot from 2023-02-14 14-10-25](https://user-images.githubusercontent.com/83389170/218684154-5335d424-ca8d-4537-b34c-1f1f9291056e.png)
can someone help me with this.

**Comment by timuric** [comment]
Right now it is not possible to search by type, but we are going to replace this modal with proper filtering functionality

**Comment by ghost** [comment]
@timuric do we have an estimate when proper filtering functionality can be expected ?
